Create README.md

rbeyer opened the pull request:
This is to add a link for our shiny new Google Calendar.  I still need individual addresses to give permissions for others to add events to the calendar.

rbeyer committed:
Create README.md

This is to add a link for our shiny new Google Calendar.  I still need individual addresses to give permissions for others to add events to the calendar.

jessemapel reviewed (commented):
We should try and keep the language as affirmative as possible. I'd change

> should be entered

to

> will be entered

jessemapel commented on meetings/README.md:
```diff
@@ -0,0 +1,4 @@
+As meetings are planned above, they should be entered into our
+[Google Calendar](https://calendar.google.com/calendar?cid=aXFtYmE4MmQ2MGUwdDRycGIzZ2U0MjhwbTBAZ3JvdXAuY2FsZW5kYXIuZ29vZ2xlLmNvbQ)
+that you can subscribe to.  If you are a TSC member and need the ability to post events, send Ross your address to give permissions to.
```
Giving people permissions to the calendar should be addressed under the [onboarding document](https://github.com/planetarysoftware/TSC/blob/master/ONBOARDING.md)

rbeyer replied:
There only, or both places?

jessemapel replied:
I would say there only, the readme should be for the general viewing public.

rbeyer committed:
Updated

Took Jesse's suggestions about wording and content, and added a general 'all are welcome' statement.
